### PR TITLE
fixed CVE issues reported by sonatype ossindex audit

### PR DESCRIPTION
**Pull request by pj-spoelders**

### CODE REVIEW

File: doc/changes/changes_1.3.1.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Parquet for Java 1.3.1, released 2022-06-07
+
+Code name: Dependency updates / CVE fixes
+
+## Summary
+
+Dependency updates
```
Comment (tomuben) [inline doc/changes/changes_1.3.1.md:7]: I remember asked us to use more specific names for the release summary. Maybe better:
```suggestion
Compile dependency updates for guava, hadoop-client and parquet-hadoop, along with updates for test and plugin packages.
```

**Review by tomuben — COMMENTED** [review]
Please check my comment.

File: doc/changes/changes_1.3.1.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Parquet for Java 1.3.1, released 2022-06-07
```
Comment (kaklakariada) [inline doc/changes/changes_1.3.1.md:1]:
```suggestion
# Parquet for Java 1.3.1, released 2022-06-08
```

**Review by kaklakariada — CHANGES_REQUESTED** [review]
Also replace `release_config.yml` with this new content:

```yml
release-platforms:
  - GitHub
  - Maven
  - Jira
```

**Comment by kaklakariada** [comment]
@pj-spoelders : I overlooked that you already made this PR and created #43. Sorry ☹️

**Pull request closed without merging**